src/components/Slider/SliderBase/SliderBase.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { forwardRef } from "react";
+import PropTypes from "prop-types";
+import "./SliderBase.scss";
+import { useSliderActions, useSliderSelection, useSliderUi } from "../SliderContext";
+import { bem, calcDimensions, getNearest, moveToPx } from "../SliderHelpers";
+import { useSliderRail } from "../SliderHooks";
+import SliderRail from "./SliderRail";
+import SliderTrack from "./SliderTrack";
+import SliderFilledTrack from "./SliderFilledTrack";
+import SliderThumb from "./SliderThumb";
+import {
+  isArrowDownEvent,
+  isArrowLeftEvent,
+  isArrowRightEvent,
+  isArrowUpEvent,
+  isPageDownEvent,
+  isPageUpEvent,
+  isEndEvent,
+  isHomeEvent
+} from "../../../utils/dom-event-utils";
+
+function getKey(index) {
```

Comment:
why ? it is used once  and it is the identity function

Reply:
Looks like missed it after some changes/refactoring 🤦🏻‍♂️ 😆
Good catch!
Changing....

Reply:
Done!